### PR TITLE
Tranlate - guide - migration - events

### PR DESCRIPTION
**Pull request by Tintef**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -765,38 +765,38 @@ Si necesita escalonar las transiciones de la lista, puede controlar el tiempo co
...
 {% endraw %}
 
-## Events
+## Eventos
 
-### `events` option <sup>removed</sup>
+### Opción `events` <sup>removido</sup>
 
-The `events` option has been removed. Event handlers should now be registered in the `created` hook instead. Check out the [`$dispatch` and `$broadcast` migration guide](#dispatch-and-broadcast-replaced) for a detailed example.
+La opción `events` ha sido removida. Ahora, Los manejadores de eventos deben ser registrados en el _hook_ `created`. Vea la [guía de migración de `$dispatch` y `$broadcoast`](#dispatch-and-broadcast-replaced) para un ejemplo detallado.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
La opción `events` ha sido removida. Ahora, Los manejadores de eventos deben ser registrados en el _hook_ `created`. Vea la [guía de migración de `$dispatch` y `$broadcast`](#dispatch-and-broadcast-replaced) para un ejemplo detallado.
```

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -765,38 +765,38 @@ Si necesita escalonar las transiciones de la lista, puede controlar el tiempo co
...
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos de la vieja sintaxis de configuración de <code>keyCode</code>.</p>
 </div>
 {% endraw %}
 
-### `$dispatch` and `$broadcast` <sup>replaced</sup>
+### `$dispatch` y `$broadcast` <sup>reemplazado</sup>
 
-`$dispatch` and `$broadcast` have been removed in favor of more explicitly cross-component communication and more maintainable state management solutions, such as [Vuex](https://github.com/vuejs/vuex).
+`$dispatch` and `$broadcast` han sido removidos en favor de una comunicación más explícita entre componentes y soluciones de manejo de estado más mantenibles, así como [Vuex](https://github.com/vuejs/vuex).
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
`$dispatch` y `$broadcast` han sido removidos en favor de una comunicación más explícita entre componentes y soluciones de manejo de estado más mantenibles, así como [Vuex](https://github.com/vuejs/vuex).
```

File: src/v2/guide/migration.md [hunk 3]
```diff
@@ -765,38 +765,38 @@ Si necesita escalonar las transiciones de la lista, puede controlar el tiempo co
...
 
-The problem is event flows that depend on a component's tree structure can be hard to reason about and very brittle when the tree becomes large. It doesn't scale well and we don't want to set you up for pain later. `$dispatch` and `$broadcast` also do not solve communication between sibling components.
+El problema es los flujos de eventos que dependen de la estructura del árbol de componentes pueden ser difíciles de razonar y muy fráfiles cuando el árobl se vuelve grande. No escala bien y no queremos prepararnos para sufrir luego. `$dispatch` y `$broadcast` tampoco resuelven la comunicación entre componentes hermanos.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
El problema son los flujos de eventos que dependen de la estructura del árbol de componentes, pueden ser difíciles de razonar y muy frágiles cuando el árbol se vuelve grande. No escala bien y no queremos prepararnos para sufrir luego. `$dispatch` y `$broadcast` tampoco resuelven la comunicación entre componentes hermanos.
```

File: src/v2/guide/migration.md [hunk 4]
```diff
@@ -765,38 +765,38 @@ Si necesita escalonar las transiciones de la lista, puede controlar el tiempo co
...
 
-One of the most common uses for these methods is to communicate between a parent and its direct children. In these cases, you can actually [listen to an `$emit` from a child with `v-on`](components.html#Form-Input-Components-using-Custom-Events). This allows you to keep the convenience of events with added explicitness.
+Uno de los casos de uso más comúnes para estos métodos es para la comunicación directaentre un componente padre y sus hijos. En estos casos, usted pueden [escuchar a un `$emit` desde un componente hijo con `v-on`](components.html#Form-Input-Components-using-Custom-Events). Esto le permite mantener la conveniencia de los eventos.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
Uno de los casos de uso más común para estos métodos es para la comunicación directa entre un componente padre y sus hijos. En estos casos, usted pueden [escuchar a un `$emit` desde un componente hijo con `v-on`](components.html#Form-Input-Components-using-Custom-Events). Esto le permite mantener la conveniencia de los eventos.
```

File: src/v2/guide/migration.md [hunk 5]
```diff
@@ -765,38 +765,38 @@ Si necesita escalonar las transiciones de la lista, puede controlar el tiempo co
...
 
-However, when communicating between distant descendants/ancestors, `$emit` won't help you. Instead, the simplest possible upgrade would be to use a centralized event hub. This has the added benefit of allowing you to communicate between components no matter where they are in the component tree - even between siblings! Because Vue instances implement an event emitter interface, you can actually use an empty Vue instance for this purpose.
+Sin embargo, cuando se quiere comunicar descendientes/antepasados distantes, `$emit` no le será de ayuda. En cambio, la forma más simple de resolver este problema sería utilizando un _event hub_ centralizado. Esto tiene la ventaja de que permite comunicar componente sin importar dónde esten en el árbol de componentes - ¡incluso si son hermanos! Dado que las instancias de Vue implementar una interfaz de emisión de eventos, usted puede utilizar una instancia de Vue vacía para este propósito.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
Sin embargo, cuando se quiere comunicar descendientes/antepasados distantes, `$emit` no le será de ayuda. En cambio, la forma más simple de resolver este problema sería utilizando un _event hub_ centralizado. Esto tiene la ventaja de que permite comunicar componente sin importar dónde estén en el árbol de componentes - ¡incluso si son hermanos! Dado que las instancias de Vue implementan una interfaz de emisión de eventos, usted puede utilizar una instancia de Vue vacía para este propósito.
```

File: src/v2/guide/migration.md [hunk 6]
```diff
@@ -805,15 +805,15 @@ Todos
...
 ```
 
-We could manage communication between components with this single event hub:
+Podríamos manejar la comunicación entre componentes con un simple _event hub_:
 
 ``` js
-// This is the event hub we'll use in every
-// component to communicate between them.
+// Este es el event hub que usaremos en cada
+// componente para comunicarlas entre sí.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
// componente para comunicarlos entre sí.
```

File: src/v2/guide/migration.md [hunk 7]
```diff
@@ -843,8 +843,8 @@ created: function () {
   eventHub.$on('add-todo', this.addTodo)
   eventHub.$on('delete-todo', this.deleteTodo)
 },
-// It's good to clean up event listeners before
-// a component is destroyed.
+// Es bueno limpiar los listeners antes
+// de que una componente sea destruida.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
// de que un componente sea destruido.
```

File: src/v2/guide/migration.md [hunk 8]
```diff
@@ -861,12 +861,13 @@ methods: {
 }
 ```
 
-This pattern can serve as a replacement for `$dispatch` and `$broadcast` in simple scenarios, but for more complex cases, it's recommended to use a dedicated state management layer such as [Vuex](https://github.com/vuejs/vuex).
+Este patrón puede servir como reemplazo para `$dispatch` y `$broadcast` en escensario simples, pero para casos más complejos, se recomienda el uso de una capa dedicada al manejo de estado, tal como [Vuex](https://github.com/vuejs/vuex).
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
Este patrón puede servir como reemplazo para `$dispatch` y `$broadcast` en escenarios simples, pero para casos más complejos, se recomienda el uso de una capa dedicada al manejo de estado, tal como [Vuex](https://github.com/vuejs/vuex).
```

**Review by sxcamacho — CHANGES_REQUESTED** [review]

**Comment by Tintef** [comment]
@sxcamacho Thanks for the review! All of them updated 😄

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**